recipes/ckiptagger/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ckiptagger" %}
+{% set version = "0.0.19" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/aa/ca/aa47ec77ae30d8068dc76f4dbf52bb48290e012dbe4a0c96b4feaa0f6dc8/ckiptagger-0.0.19-py3-none-any.whl"
```

Comment:
is a source package not included? conda-forge typically prefers building from source.

Reply:
if i used source code
its not working on windows

Reply:
Do you have a link to a build? if not, ideally this would be contained in different PR so we can focus on that issue idependently,